### PR TITLE
[subdevice] items_optional for Color temperature light

### PR DESCRIPTION
**Pull request by Zehir**
Part 9 of https://github.com/dresden-elektronik/deconz-rest-plugin/pull/7155

### CODE REVIEW

File: devices/generic/subdevices/color_temperature_light.json [hunk 1]
```diff
@@ -6,22 +6,26 @@
...
+        "state/bri",
+        "state/colormode",
+        "state/ct",
+        "state/on",
+        "state/reachable"
+    ],
+    "items_optional":[
         "cap/alert/trigger_effect",
         "cap/bri/move_with_onoff",
         "cap/color/capabilities",
```
Comment (ebaauw) [inline devices/generic/subdevices/color_temperature_light.json:21]: `cap/color/capabilities` should be mandatory.

**Review by ebaauw — CHANGES_REQUESTED** [review]
See also https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Mandatory-and-Optional-Items-for-Lights-Resources.

File: devices/generic/subdevices/color_temperature_light.json [hunk 2]
```diff
@@ -6,22 +6,26 @@
...
-        "cap/color/ct/max",
-        "cap/color/ct/min",
+        "cap/groups_not_supported",
```
Comment (ebaauw) [inline devices/generic/subdevices/color_temperature_light.json:22]: `cap/groups/not_supported`

File: devices/generic/subdevices/color_temperature_light.json (hunk 1 above)
Comment (Zehir) [reply devices/generic/subdevices/color_temperature_light.json:21]: Done

File: devices/generic/subdevices/color_temperature_light.json (hunk 2 above)
Comment (Zehir) [reply devices/generic/subdevices/color_temperature_light.json:22]: Done

**Review by ebaauw — APPROVED** [review]

**Pull request merged**